### PR TITLE
Write seed scripts for csv and json

### PR DESCRIPTION
**Pull request by jscholar**
JSON seed script is formatted by 'lines'.
10M game_ids each. Number of updates varies.

### CODE REVIEW

File: database/scripts/seedUpdateDocs.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        if (!outFile.write(doc)) {
+            await new Promise((resolve) => {
+                outFile.once('drain', resolve);
+            });
+        }
+        
+    }
+}
+
+writeGameUpdates();
```
Comment (phonxaychampasouk) [inline database/scripts/seedUpdateDocs.js:46]: Looks like a new line is missing

File: database/scripts/seedUpdateRecs.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            ]
+            const csv = record.join(',') + '\n';
+            if (!outFile.write(csv)) {
+                await new Promise((resolve) => {
+                    outFile.once('drain', resolve);
+                });
+            };
+        }
+    }
+}
```
Comment (phonxaychampasouk) [inline database/scripts/seedUpdateRecs.js:35]: New Line.

File: database/scripts/schema.sql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+CREATE TABLE updates (
+    game_id         integer NOT NULL,
+    posted_by       char(100),
+    post_date       char(100),
+    title           char(100),
+    text            text,
+    img             text,
+    comment_count   integer
+);
```
Comment (phonxaychampasouk) [inline database/scripts/schema.sql:9]: Everything looks good man. Except for those lines missing.

File: database/scripts/seedUpdateRecs.js (hunk 2 above)
Comment (jscholar) [reply database/scripts/seedUpdateRecs.js]: Fixed with 64393ba

File: database/scripts/seedUpdateDocs.js (hunk 1 above)
Comment (jscholar) [reply database/scripts/seedUpdateDocs.js]: Fixed with 64393ba

File: database/scripts/schema.sql (hunk 3 above)
Comment (jscholar) [reply database/scripts/schema.sql:9]: Ok, I fixed them. Thanks. 
64393ba

**Pull request merged**